modules/schema/src/main/java/org/apache/ignite/internal/schema/marshaller/schema/CalcSizeByteBuffer.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.schema.marshaller.schema;
+
+/**
+ * ExtendedByteBuffer implementation that does not store any data but counts the size of the data that passed through it.
+ */
+public class CalcSizeByteBuffer implements ExtendedByteBuffer {
```

Comment:
This class looks useless.

Reply:
Removed group of ExtendedByteBuffer classes.